transformer/dockerfilegenerator/dotnetcoredockerfilegenerator.go

```diff
@@ -39,7 +39,7 @@ import (
 
 var (
 	portRegex            = regexp.MustCompile(`:(\d+)`)
-	dotNetCoreFrameworks = []string{"net5.0", "netcoreapp2.1", "netstandard2.0"}
+	dotNetCoreFrameworks = []string{"net5.0", "netcoreapp2.1", "netstandard2.0", "netcoreapp3.1"}
```

Comment:
@Akash-Nayak As discussed, lets try to use a regex or some other way so that we can cover the full spectrum of versions, rather than trying to support each version individually.